dist/backdraft.js

```diff
@@ -1275,8 +1275,18 @@ _.extend(Plugin.factory, {
 
     render : function() {
       var cells = this.findCells(), node;
-      _.each(this.columnsConfig, function(config) {
-        node = cells.filter(config.nodeMatcher(config));
+      var cleanConfigs = this.columnsConfig.map(function(currentValue,index){
```

Comment:
What does `cleanConfigs` mean?  what are you cleaning up or normalizing? Can you describe it in the variable name so the code is more clear?